docs/design/services/s3/transfermanager/README.md

```diff
@@ -0,0 +1,66 @@
+# Project Tenets (unless you know better ones)
+
+1. Meeting customers in their problem space allows them to deliver value
+   quickly.
+2. Meeting customer expectations drives usability.
+3. Discoverability drives usage.
+
+# Introduction
+
+This project provides a much improved experience for S3 customers needing to
+easily perform uploads and downloads of objects to and from S3 by providing
+the S3 `TransferManager`, a high level library built on the S3 client.
+
+# Project Goals
+
+1. For the use cases it addresses, i.e. the transfer of objects to and from S3,
+   TransferManager is the preferred solution. It is easier and more intuitive
+   than using the S3 client. In the majority of situations, it is more
+   performant.
+1. TransferManager provides a truly asynchronous API that conforms to the norms
+   present in the rest of the SDK.
```

Comment:
Non-blocking, I assume?

Reply:
Yes, good catch. I'll update the wording